lib/github.js

```diff
@@ -71,6 +71,12 @@ Organization.prototype.getRepositories = function(callback) {
             cli.debug('next page ' + this.nextPageUrl);
             cli.debug('last page ' + this.getLastPage());
 
+            if ( body.message ) {
```

Comment:
Are you sure we can remove this part? It seems that lib returns 200 in case of GitHub errors so that check for `message` presence will be still needed